### ISSUE TITLE
How to use the SDK client hfc to unregister a user?

### ISSUE THREAD

**Issue by huhaiqing** [hyperledger-archives/fabric#2269]
Hi,

Currently, I only find the API to register a user, chain.registerAndEnroll(registrationRequest,cb);

Anyone can help to tell me how to unregister the user which registered by chain.registerAndEnroll( ...) function.

Thanks in advance.

Regards
Harry

**Comment by smithbk** [comment]
Hi, unfortunately it is not currently supported.  See https://github.com/hyperledger/fabric/issues/1620

**Comment by huhaiqing** [comment]
@smithbk , Hi, Is there any way to remove the registered user mannually? Thanks

**Comment by smithbk** [comment]
No, unfortunately not for a single user.  The only thing you can do is remove all users except those pre-registered (but not enrolled) in the yaml file.
To do this ...
On the membership services side:
  1) stop membership services
  2) rm -rf /var/hyperledger/production
  3) start membership services
On the client SDK side:
    rm -rf /tmp/keyValStore

**Comment by gromeroar** [comment]
hi @srderson could you assign to me this issue? Thanks!!
